src/toxcore/dht/server/mod.rs

```diff
@@ -422,17 +429,36 @@ impl Server {
             )
         }
 
+        let mut futures = Vec::new();
+
+        let addrs = target_peer.get_all_sockets(self.is_ipv6_enabled);
+
+        addrs.into_iter()
+            .for_each(|addr| {
+                let ping_id = request_queue.new_ping_id(target_peer.pk);
+                futures.push(self.send_nodes_req_inner(addr, target_peer.pk, search_pk, ping_id));
+            });
+
+        if futures.is_empty() {
```

Comment:
Now we can move this warning to `get_all_sockets` so that any usage of this method will make it.